app/models/course/announcement.rb

```diff
@@ -18,7 +17,7 @@ def to_partial_path
 
   private
 
-  def send_notification
-    Course::AnnouncementNotifier.new_announcement(creator, self)
-  end
+  # Override this function from the ReminderConcern as we don't want closing reminders
+  # from announcements
+  def setup_closing_reminders; end
```

Comment:
I think you shouldn't be overwriting the concern's method like that. If there is an unneeded method, split the reminder concern into 2 and just include what is necessary.

Reply:
Fixed, but don't merge yet, need to run some manual tests to make sure everything still works.